convert2rhel/backup/subscription.py

```diff
@@ -71,11 +73,15 @@ def __init__(self):
         self._is_attached = False
 
     def enable(self):
+        if self.enabled:
```

Comment:
I see codecov is complaining about this not tested, maybe we should cover this? But it's so straight forward...

Reply:
Yes it should be covered